react/features/welcome/components/WelcomePage.web.js

```diff
@@ -214,14 +214,16 @@ class WelcomePage extends AbstractWelcomePage {
                             </form>
                         </div>
                         <div
-                            className = 'welcome-page-button'
+                            className = 'welcome-page-button welcome-page-button-go'
```

Comment:
I thought `welcome-page-button` should be left as it is to avoid something from breaking.